files/bootstrap.sh

```diff
@@ -51,6 +52,11 @@ while [[ $# -gt 0 ]]; do
             shift
             shift
             ;;
+        --enable-docker-bridge)
+            ENABLE_DOCKER_BIDGE=$2
```

Comment:
There is a typo here, I just spent an hour debugging this! Can you please release a hotfix for this?

Reply:
@micahhausler @nckturner

Reply:
Yeah, this is a bit unfortunate.